src/ingest_validation_tools/table-schemas/level-1.yaml

```diff
@@ -44,33 +44,41 @@ fields:
     description: 'The specific type of assay being executed.'
     constraints:
       enum:
-        - scRNA-Seq (10xGenomics)
-        - AF
-        - bulk RNA
-        - bulkATACseq
+        - Autofluorescence Microscopy
+        - Bulk ATAC-seq
         - CODEX
         - Imaging Mass Cytometry
-        - LC-MS (metabolomics)
-        - LC-MS/MS (label-free proteomics)
-        - MxIF
-        - MALDI-IMS positive
-        - MALDI-IMS negative
-        - MS (shotgun lipidomics)
-        - PAS microscopy
-        - scATACseq
-        - sciATACseq
-        - sciRNAseq
+        - Untargeted LC-MS
```

Comment:
Labeled LC-MS/MS

Reply:
`Untargeted LC-MS` is not in current version.